### ISSUE TITLE
Add User

### ISSUE THREAD

**Issue by romzungab** [dansalan/DCFIv3#2]
Mam kani ang kulang sa AddUser
1. check valid inputs
2. password and retype password checking
3. Message/Feedback

**Comment by RazenRyne** [comment]
Unsa ning message or feedback mam? Katoh ng mga error message?

Sent from Yahoo Mail on Android

**Comment by romzungab** [comment]
Yes mam..error messages or confirmation messages.. ang ref  message nga parameter sa service kay pwede na gamiton nga message mam

**Comment by RazenRyne** [comment]
Okz mam

Sent from Yahoo Mail on Android
